package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "goatcounter",
+  "version": "4.0.0",
+  "description": "Easy web analytics. No tracking of personal data.",
+  "keywords": [
+	"go",
+	"golang",
+	"analytics"
+  ],
+  "main": "public/count.js",
+  "types": "count.d.ts",
+  "scripts": {
+    "prepack": "cp README.md git.README.md && mv README.npm.md README.md",
+	"postpack": "mv README.md README.npm.md && mv git.README.md README.md"
```

Comment:
This temporarily swaps the repository's `README.md` with `README.npm.md` during the npm packing process such that the npm package can use a different, slimmed down README.